velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -501,3 +502,36 @@ TEST_F(PlanNodeToStringTest, tableScan) {
         plan->toString(true, false));
   }
 }
+
+TEST_F(PlanNodeToStringTest, window) {
+  std::vector<exec::FunctionSignaturePtr> signatures{
+      exec::FunctionSignatureBuilder()
+          .argumentType("BIGINT")
+          .returnType("BIGINT")
+          .build(),
+  };
+  exec::registerWindowFunction("window1", std::move(signatures), nullptr);
+
+  auto plan =
+      PlanBuilder()
+          .tableScan(ROW({"a", "b", "c"}, {VARCHAR(), BIGINT(), BIGINT()}))
+          .window({"window1(c) over (partition by a order by b "
+                   "range between 10 preceding and unbounded following) AS d"})
+          .planNode();
+  ASSERT_EQ(
+      "-- Window[partition by: [a] order by: [b ASC NULLS LAST] "
+      "d := window1(ROW[\"c\"]) RANGE BETWEEN 10 PRECEDING AND UNBOUNDED FOLLOWING] "
+      "-> a:VARCHAR, b:BIGINT, c:BIGINT, d:BIGINT\n",
+      plan->toString(true, false));
```

Comment:
perhaps, add tests for plan->toString() as well

Reply:
Please can you clarify further. All PlanNode testing that is specific to different plan nodes look at details only at that level. There are other tests for recursive plans. Did you have any other tests in mind ?

Reply:
Other tests, first verify that toString() without parameters returns a plan without details. I was thinking that it would be nice to add similar test for window.

```
TEST_F(PlanNodeToStringTest, orderBy) {
  auto plan = PlanBuilder()
                  .values({data_})
                  .orderBy({"c1 ASC NULLS FIRST"}, true)
                  .planNode();

  ASSERT_EQ("-- OrderBy\n", plan->toString());
```